src/services/shared-characters.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  ApplicationCommandOptionChoiceData,
+  GuildMemberRoleManager,
+} from "discord.js";
+import { some, truncate } from "lodash";
+import { Account, getAccounts } from "./spreadsheets/accounts";
+import {
+  getBots,
+  updateBotCheckoutTime,
+  updateBotLocation,
+  updateBotPilot,
+} from "./spreadsheets/bots";
+import moment from "moment";
+import { getParkLocations } from "./spreadsheets/park-locations";
+
+export const sharedCharacters = {
```

Comment:
This is the service entrypoint for working with shared characters. Put helper utilities here.

The actual wrappers into the spreadsheets are abstracted out to keep this simple and focused on logic not read/write.